core/commands/flow.py

```diff
@@ -0,0 +1,325 @@
+import sublime
+from sublime_plugin import WindowCommand
+
+from ..git_command import GitCommand
+
+
+INIT_REQUIRED_MSG = "Please run `git: flow init` first"
+
+GITFLOW_CONF = ['branch.master', 'branch.develop', 'prefix.feature',
+                'prefix.release', 'prefix.hotfix', 'prefix.versiontag',
+                'prefix.support', 'origin', ]
+
+
+class GsFlowCommon(WindowCommand, GitCommand):
```

Comment:
Add a doc-string here to the class, explaining at a high level what the command does.